src/main.rs

```diff
@@ -77,6 +77,8 @@ fn main() {
     let port = config.port;
     let max_filesize = (config.max_filesize as f64 * 1.37) as usize;
 
+    println!("Listening on port {}", port);
```

Comment:
Printing to stderr would probably be more appropriate here

Reply:
How come, It isn’t an error? I can make the change tomorrow but you do them now can if you like.

Reply:
I've just been taught to use stderr for logs and notices, so that's more of a personnal preference,

Reply:
But yeah thinking of it it might make more sense to use stdout for that specific message since it's an expected startup message.